### PR TITLE
YDA-5699 add admin page

### PR DESCRIPTION
**Pull request by FuHsinyu**
YDA-5699
Added the blueprint admin.py and template/index.html for an empty admin page.
Added CSS file in the static folder for the admin page

### CODE REVIEW

File: admin/admin.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+__copyright__ = 'Copyright (c) 2021, Utrecht University'
```
Comment (lwesterhof) [inline admin/admin.py]: 2021 -> 2024

File: admin/static/admin/css/admin.css
Comment (lwesterhof) [inline admin/static/admin/css/admin.css:1]: For now we don't need this CSS file. We will try to do all the layout with the https://getbootstrap.com/ framework, only with exception we will add custom CSS. This ensures that the whole of Yoda plays nice with themes.

File: admin/templates/admin/admin.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ super() }} &dash; Administrition{% endblock title %}
```
Comment (lwesterhof) [inline admin/templates/admin/admin.html]: Small typo

File: admin/templates/admin/admin.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+{% block style %}
+    <link rel="stylesheet" href="{{ url_for('admin_bp.index', filename='css/stats.css') }}">
```
Comment (lwesterhof) [inline admin/templates/admin/admin.html]: CSS include not needed for now.

File: general/templates/general/index.html [hunk 4]
```diff
@@ -15,4 +15,4 @@
         {% include 'index_not_authenticated.html' %}
     {% endif %}
 
-{% endblock content %}
+{% endblock content %}
```
Comment (lwesterhof) [inline general/templates/general/index.html]: Please configure Visual Studio Code insert a new line at the end of files.

**Comment by lwesterhof** [comment]
Congratulations on your first PR! 🎉

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: admin/static/admin/css/admin.css
Comment (FuHsinyu) [reply admin/static/admin/css/admin.css:1]: I'll look into that the new layout! Thanks for suggestions.

File: general/templates/general/index.html (hunk 4 above)
Comment (FuHsinyu) [reply general/templates/general/index.html]: Good idea, will do.

**Pull request merged**